.pre-commit-config.yaml

```diff
@@ -5,14 +5,10 @@ repos:
     hooks:
       - id: black
         args: [--line-length=160]
```

Comment:
is 160 really what we want? :)

Reply:
ah, btw, the default_stages at the top might not be what we'd like, @sansyrox was there a particular reason it is only set to push?

Reply:
@patrick91 

> is 160 really what we want? :)

This was just something random. I just feel that 70 characters is a bit too less :p

> ah, btw, the default_stages at the top might not be what we'd like, @sansyrox was there a particular reason it is only set to push?

@patrick91 , I am sorry but I don't understand what you mean by this. Can you please explain a bit more? 😅

Reply:
> ah, btw, the default_stages at the top might not be what we'd like, @sansyrox was there a particular reason it is only set to push?

Now, I understand. It was mistake 🤣